ui/src/components/admin/Notifications/NotificationsController.js

```diff
@@ -10,9 +10,13 @@ const NotificationsProvider = ({ location, children }) => {
   // only used for imports initiated from individual model pages
   const [importNotifications, setImportNotifications] = useState([]);
   // bulk variant import progress
-  const [importProgress, setImportProgress] = useState(DEFAULT_IMPORT_PROGRESS);
+  const [importProgress, setImportProgress] = useState(DEFAULT_PROGRESS_QUEUES);
   // models which failed variant import with nonactionable errors
   const [nonactionableImports, setNonactionableImports] = useState(DEFAULT_NONACTIONABLE_IMPORTS);
+  // only used for publishes initiated from individual model pages
```

Comment:
Similar to the `NotificationsToaster` changes, we extend NotificationsController to support and display publish notifications.